### PR TITLE
Adopt semantic-release

### PR DESCRIPTION
**Pull request by ynotdraw**
### Changes

- Adopts `semantic-release` like we have in Chroma (hat tip to @smolster for doing this in Chroma)
- Added `CODE_OF_CONDUCT` and `CONTRIBUTING` files

### CODE REVIEW

File: release.config.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+module.exports = {
+  branches: ['master'],
+  plugins: [
+    ['@semantic-release/commit-analyzer', { preset: 'conventionalcommits' }],
+    '@semantic-release/npm',
```
Comment (ynotdraw) [inline release.config.js:5]: Over in Chroma, we specify `pkgRoot`, but it requires a `package.json` file according to the docs.

> Note: The pkgRoot directory must contain a package.json. The version will be updated only in the package.json and npm-shrinkwrap.json within the pkgRoot directory.

When installing Chromicons, the package has the following contents.  Due to this, I believe we can leave this as is, but please correct me if I'm wrong here 🙇 
Screen Shot 2021-07-28 at 1 15 11 PM

Comment (smolster) [reply release.config.js:5]: If your goal is to have the published package look like the file structure you posted above, this looks great!

**Review by smolster — APPROVED** [review]
You nailed it!

**Review by HaleyWardo — APPROVED** [review]
💥

**Comment by ynotdraw** [comment]
Sorry y'all, had to rebase after versioning one last time the old way.

**Review by smolster — APPROVED** [review]

**Review by HaleyWardo — APPROVED** [review]

**Review by xdumaine — APPROVED** [review]

**Pull request merged**